CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 ### Added
 - Unions in output messages show with new syntax
+- Return type is `None` by default unless def is untyped
```

Comment:
i don't really get what this means. if a function is not typed whats its default return type?

Reply:
`Any`

Reply:
### 
```suggestion
- Default return type is `None` if `--allow-untyped-defs` is disabled, and `Any` if it's enabled
```

Reply:
forget to push?